app/src/main/res/values/dimen.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="parking_available_edit_text_width">250dp</dimen>
+    <dimen name="parking_available_edit_text_height">80dp</dimen>
+    <dimen name="fragment_tittle_text_view_text_size">16sp</dimen>
+    <dimen name="layout_marginBottom">20dp</dimen>
```

Comment:
please try to put a more representative name. remember the naming conventions https://docs.google.com/document/d/1kIL-jK55v_kWITs9qcW7d-0yJemUtLA74hQzm1_S-5k/edit?usp=sharing

Reply:
done